### PR TITLE
Improve todo-group sample

### PR DESCRIPTION
**Pull request by halter73**
The main thing this does is separate private todos from public todos.

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis.md [hunk 1]
```diff
@@ -1594,19 +1596,17 @@ app.MapGet("/text", () => Results.Text("This is some text"));
...
+app.MapGet("/pokemon", async () => 
+{
+    var stream = await proxyClient.GetStreamAsync("http://consoto/pokedex.json");
+    // Proxy the response as JSON
+    return Results.Stream(stream, "application/json");
+});
+```
 
-[!code-csharp[](~/fundamentals/minimal-apis/resultsStream/7.0-samples/ResultsStreamSample/Program.cs?name=snippet_video)]
+See <xref:fundamentals/minimal-apis/responses#stream7> for mor examples.
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis.md:1609]:
```suggestion
See <xref:fundamentals/minimal-apis/responses#stream7> for more examples.
```

File: aspnetcore/fundamentals/minimal-apis.md [hunk 2]
```diff
@@ -1646,6 +1646,8 @@ The types implementing `IResult` are public, allowing for type assertions when t
 
 You can look at the return types of the corresponding methods on the static [TypedResults](/dotnet/api/microsoft.aspnetcore.http.typedresults) class to find the correct public `IResult` type to cast to.
 
+See <xref:fundamentals/minimal-apis/responses> for mor examples.
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis.md:1649]:
```suggestion
See <xref:fundamentals/minimal-apis/responses> for more examples.
```

**Review by tdykstra — APPROVED** [review]
LGTM, just a typo to fix.

File: aspnetcore/fundamentals/minimal-apis.md (hunk 1 above)
Comment (halter73) [inline aspnetcore/fundamentals/minimal-apis.md:1609]:
```suggestion
See <xref:fundamentals/minimal-apis/responses#stream7> for more examples.
```

**Pull request merged**